front/components/assistant/GalleryAssistantPreviewContainer.tsx

```diff
@@ -1,46 +1,26 @@
 import { AssistantPreview } from "@dust-tt/sparkle";
 import type {
   LightAgentConfigurationType,
-  PlanType,
   WorkspaceType,
 } from "@dust-tt/types";
-import type { SyntheticEvent } from "react";
 
 import AssistantListActions from "@app/components/assistant/AssistantListActions";
 import { SharingChip } from "@app/components/assistant/Sharing";
-import { isLargeModel } from "@app/lib/assistant";
-import { isUpgraded } from "@app/lib/plans/plan_codes";
 
 interface GalleryAssistantPreviewContainerProps {
   agentConfiguration: LightAgentConfigurationType;
   onShowDetails: () => void;
-  onUpdate: () => void;
   owner: WorkspaceType;
-  plan: PlanType | null;
-  setTestModalAssistant?: (
-    agentConfiguration: LightAgentConfigurationType
-  ) => void;
 }
 
 export function GalleryAssistantPreviewContainer({
```

Comment:
This component is now only instantiating an `AssistantPreview`, let's ditch it no?

Reply:
I left it since @PopDaph will remove gallery in the next hours?days?